.vscode/launch.json

```
@@ -1,18 +1,44 @@
 {
     "version": "0.2.0",
     "configurations": [
-
         {
-            "name": ".NET Core Launch (console)",
-            "type": "coreclr",
+            "type": "PowerShell",
+            "request": "launch",
+            "name": "Build",
+            "script": "./build.ps1",
+            "args": [
+                ""
+            ],
+            "cwd": "${workspaceFolder}",
+            "createTemporaryIntegratedConsole": true
+        },
+        {
+            "type": "PowerShell",
+            "request": "launch",
+            "name": "Build & Run all tests",
+            "script": "./build.ps1; ./build.ps1 -Test",
+            "args": [
+                ""
+            ],
+            "cwd": "${workspaceFolder}",
+            "createTemporaryIntegratedConsole": true
+        },
+        {
+            "type": "PowerShell",
             "request": "launch",
-            "preLaunchTask": "build",
-            "program": "${workspaceRoot}/bin/Debug/<target-framework>/<project-name.dll>",
-            "args": [],
-            "cwd": "${workspaceRoot}",
-            "externalConsole": false,
-            "stopAtEntry": false,
-            "internalConsoleOptions": "openOnSessionStart"
+            "name": "PowerShell Run current Pester File",
```

Should this have a more specific name, since it imports the PSSA dev build beforehand? Maybe `Run current Pester file with PSScriptAnalyzer development build` or something?

---

OK, will apply that:
```suggestion
            "name": "Run current Pester file with PSScriptAnalyzer development build",
```